blueprints/controller/files/__root__/__path__/__name__.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Controller definition for the <%= dasherizedModuleName %> controller
```

Comment:
Also, I happen to find it useful when looking at multiple files, all called `controller.js` to have some context as to which controller I'm looking at.